scripts/jenkins/ardana/ansible/roles/ardana_tempest/tasks/reconfigure_neutron.yml

```diff
@@ -21,3 +21,14 @@
       -e '{{ ardana_extra_vars|to_json }}'
   args:
     chdir: "{{ ardana_scratch_path }}"
+
+# FIXME: workaround for SCRD-9144
+- name: Wait a while to give neutron agents time to recover
+  pause:
+    minutes: 5
```

Comment:
Nope because the status check is checking the systemctl status, which reports that the service is up and running, but in fact the service isn't ready yet... So just checking the status isn't enough.

Reply:
We'd need a condition for the `until` part, which we don't have in this case (see Fergal's comment above).

Reply:
@aojea We'd need a condition for the until part, which we don't have in this case (see Fergal's comment above).

Reply:
is not possible to use something like?
```
- shell: ansible-playbook neutron-status.yml
  register: results
  until: results | success
  retries: 10
  delay: 3
```